user_manual/security.rst

```diff
@@ -0,0 +1,18 @@
+Security
+========
+
+Passcode Locks & Pins
+---------------------
+
+You can also set a passcode lock to further protect your files and folders.
```

Comment:
Fingerprint should be included here as well.

Reply:
Just added it, @davigonz. Can you check that it's correct?